packages/perseus/src/widgets/radio/radio-validator.ts

```diff
@@ -21,7 +21,7 @@ function radioValidator(
         };
     }
 
-    // TODO: should numCorrect actually be on the rubric
+    // TODO(LEMS-2541): should numCorrect actually be on the rubric
```

Comment:
Thanks for filing this ticket. I think `numCorrect`, `noneOfTheAboveIndex`, and `noneOfTheAboveSelected` are all problematic, but I don't think we have to address it in this PR. It should probably be a fast follow though.